prism/src/test/java/org/apache/falcon/service/EntitySLAAlertServiceTest.java

```diff
@@ -172,6 +172,7 @@ public static void processSLACandidateProcess() throws FalconException, Interrup
 
 
         Thread.sleep(10*1000);
+        // process is not scheduled.
         Assert.assertTrue(monitoringJdbcStateStore.getEntityAlertInstance("test-process", "test-cluster", dateOne,
```

Comment:
I will revert this change . Its supposed to throw exception as per the code. I will debug through this.